docs/tests/integration/test_agentd/test_agentd_parametrized_reconnections.md

```diff
@@ -1,3 +1,37 @@
+# Test AgentD parametrized reconnection
+
+## Overview
+
+These tests will check that the connection is finally made when
+there are delays between connection attempts to the server.
+
+## Objective
+
+The objective is to check how AgentD behaves when there are delays between connection attempts
+to RemoteD using `TCP` and `UDP` protocols.
+
+## General info
+
+|Tier | Platforms | Number of tests | Time spent |
+|:--:|:--:|:--:|:--:|
+| 0 | Linux/Windows | 9 | 18m 39s |
+
+## Expected behavior
+
+Success if the agent connects with the manager, failure otherwise.
+
+## Testing
+
+The tests are based on testing a different number of connection attempts and time between
+them and then, verifying that the connection between AgentD and RemoteD is finally successful. 
```

Comment:
```suggestion
them and then, verify if the connection between wazuh-agentd and wazuh-remoted is finally successful. 
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca